certified-connectors/Secib/apiProperties.json

```diff
@@ -0,0 +1,55 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "{Application (client) ID of the connector in Azure App Registrations}",
+          "scopes": [
+            "User.Read",
+            "User.ReadBasic.All",
+            "api://{Application (client) ID of the web API in Azure App Registrations}/create",
```

Comment:
@thomasBeckSecib Are you sure these scope values are valid as no scope value depends on its client or application id.

Reply:
You were right, the clientId is not necessary. The scopes are now valid.